apps/projects/app/components/Panel/ReviewApplication/ReviewApplication.js

```diff
@@ -22,245 +21,185 @@ import { ipfsAdd } from '../../../utils/ipfs-helpers'
+    applicationDate: request.applicationDate
   }
 
-  changeRequest = (index) => {
-    this.setState({ requestIndex: index })
-  }
-
-  render() {
-    const { issue } = this.props
-    const request = issue.requestsData[this.state.requestIndex]
-    const application = {
-      user: {
-        login: request.user.login,
-        name: request.user.login,
-        avatar: request.user.avatarUrl,
-        url: request.user.url
-      },
-      workplan: request.workplan,
-      hours: request.hours,
-      eta: (new Date(request.eta)).toLocaleDateString(),
-      applicationDate: request.applicationDate
-    }
-
-    const applicant = application.user
-    const applicantName = applicant.name ? applicant.name : applicant.login
-    const applicationDateDistance = formatDistance(new Date(application.applicationDate), new Date())
-    return (
-      <div>
-        <IssueTitle>{issue.title}</IssueTitle>
-
-        <SafeLink
-          href={issue.url}
-          target="_blank"
-          style={{ textDecoration: 'none', color: '#21AAE7' }}
-        >
-          <IssueLinkRow>
-            <IconGitHub color="#21AAE7" width='14px' height='14px' />
-            <Text style={{ marginLeft: '6px' }}>{issue.repo} #{issue.number}</Text>
-          </IssueLinkRow>
-        </SafeLink>
-
-        <FieldTitle>Applicant</FieldTitle>
-        <DropDown
-          name="Applicant"
-          items={issue.requestsData.map( request => request.user.login)}
-          onChange={this.changeRequest}
-          active={this.state.requestIndex}
-          wide
-        />
+  const applicant = application.user
+  const applicantName = applicant.name ? applicant.name : applicant.login
+  const applicationDateDistance = formatDistance(new Date(application.applicationDate), new Date())
 
-        <ApplicationDetails>
-          <UserLink>
-            <img
-              alt=""
-              src={applicant.avatar}
-              style={{
-                width: '32px',
-                height: '32px',
-                marginRight: '10px',
-              }}
-            />
-            <SafeLink
-              href={applicant.url}
-              target="_blank"
-              style={{ textDecoration: 'none', color: '#21AAE7', marginRight: '6px' }}
-            >
-              {applicantName}
-            </SafeLink>
+  return (
+    <div css={`margin: ${2 * GU}px 0`}>
+      <Text.Block size="xlarge">{issue.title}</Text.Block>
+
+      <Link
+        href={issue.url}
+        target="_blank"
+        style={{ textDecoration: 'none', color: `${theme.link}` }}
+      >
+        <IssueLinkRow>
+          <IconGitHub color={`${theme.link}`} width='14px' height='14px' />
+          <Text css="margin-left: 6px">
+            {issue.repo} #{issue.number}
+          </Text>
+        </IssueLinkRow>
+      </Link>
+
+      <FieldTitle>Applicant</FieldTitle>
+      <DropDown
+        name="Applicant"
+        items={issue.requestsData.map( request => request.user.login)}
```

Comment:
Our linter should probably squawk about those unbalanced parentheses 😄

Reply:
Removed that extra space (sometimes linter forgives).